src/fastaudio/functional.py

```diff
@@ -0,0 +1,251 @@
+import torch
```

Comment:
This file should be inside the augment/ folder, as it contains the functional implementation of the data augmentations